build/build.cmd

```diff
@@ -0,0 +1,12 @@
+IF "%buildConfiguration%"=="" (SET configuration=Debug) ELSE (SET configuration=%buildConfiguration%)
```

Comment:
Add `setlocal` to the beginning so the env var doesn't leak to the shell (i.e. with setlocal its scope will be limited to the script)